test/functional/api/server_notifications_spec.lua

```diff
@@ -78,6 +79,9 @@ describe('notify', function()
   end)
 
   it('cancels stale events on channel close', function()
+    if uname() == 'freebsd' then
```

Comment:
TODO / off topic: could make `is_os('...bsd')` work

Reply:
Hmm, yeah could work. I'm not against it, but let's save it for a future discussion like you said.